storage/innobase/xtrabackup/doc/source/howtos/recipes_xbk_partition.rst

```diff
@@ -0,0 +1,110 @@
+================================================================================
+ Backing Up and Restoring Individual Partitions
+================================================================================
+
+|Percona XtraBackup| lets you backup
+individual partitions because partitions are regular tables with specially formatted names. The only
+requirement for this feature is having the :term:`innodb_file_per_table` option
+enabled in the server.
+
+There is only one caveat about using this kind of backup: you can not copy back
+the prepared backup. Restoring partial backups should be done by importing the
+tables. 
+
+Creating the backup
+================================================================================
+
+There are three ways of specifying which part of the whole data will be backed
+up: regular expressions (:option: --tables`), enumerating the
+tables in a file (:option:`--tables`) or providing a list of
+databases (:option:` --databases`).
+
+The regular expression provided to this option will be matched against the fully
+qualified database name and table name, in the form of
+``database-name.table-name``.
+
+For example, this operation takes a back up of the partition ``p4`` from the table ``name`` located in the database ``imdb``::
+
+  $ xtrabackup --tables=^imdb[.]name#p#p4 --backup 
```

Comment:
we need to mention PXB can't generate cfg file if partition 0 is not backed up. MySQL 8.0 stores the metadata of table in partition 0 which is required to generate cf file. 

otherwise we will get error like 
xtrabackup: export option is specified.
xtrabackup: error: cannot find dictionary record of table imdb/name#p#p4

Reply:
added.